src/chi-vue/src/views/ExpansionPanelView.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div id="expansion-panel-view">
+    <h2>Expansion Panel</h2>
+    <h3>Pending</h3>
+    <ExpansionPanel
+      v-for="(state, index) in states"
+      :key="index"
+      :step="index + 1"
+      title="Title"
+      :state="active === index ? 'active' : active > index ? 'done' : 'pending'"
+    >
+      <div slot="active">
+        <div class="chi-epanel__subtitle">
+          Optional subtitle
+        </div>
+        <p class="chi-epanel__text">
+          Content in expansion panel (e.g. a form to select a product package)
+        </p>
+      </div>
+      <div slot="done">
+        Use this area to present<br />
+        a read-only summary of what the user<br />
+        entered or selected in step 1.<br />
+        (e.g.) a package selection
+      </div>
+      <!-- <div slot="footer" class="-w--100">
+        <div class="-bg--primary">
+          Test
+        </div>
+      </div> -->
+      <button slot="footerStart" class="chi-button" @click="active -= 1">Previous</button>
+      <button slot="footerEnd" class="chi-button -primary" @click="active += 1">
```

Comment:
Does it jump to the next pending panel if I click Continue in a done step?

Reply:
`ExpansionPanelView.vue` Is a Development component not distributed in production, the functionality logic here is just for testing.